exercises/sum-of-multiples/example.go

```diff
@@ -1,16 +1,15 @@
 package summultiples
 
-func MultipleSummer(divisors ...int) func(int) int {
-	return func(limit int) int {
-		sum := 0
-		for i := 1; i < limit; i++ {
-			for _, d := range divisors {
-				if i%d == 0 {
-					sum += i
-					break
-				}
+// MultipleSummer returns the sum of the multiples of the given divisors
+// up to, but not including, the given limit.
+func MultipleSummer(limit int, divisors ...int) (sum int) {
```

Comment:
it looks like historically it made sense to call this `MultipleSummer` because it would be used like `MultipleSummer(3, 5).sum(4)` or something.

At this point, now it is being used like `MultipleSummer(4, 3, 5)`, in which case I think it is better to give it a verb name than a noun name. I am thinking of `SumMultiples`. `go lint` may complain about `summultiples.SumMultiples` being redundant, but maybe someone else has a better suggestion.

What say you? I can't find justification for this in either https://github.com/golang/go/wiki/CodeReviewComments or https://golang.org/doc/effective_go.html so it may be a personal thing, but I usually see function names as verbs.